test/functional/legacy/search_mbyte_spec.lua

```diff
@@ -0,0 +1,31 @@
+-- Test for search('multi-byte char', 'bce')
+
+local helpers = require('test.functional.helpers')
+local feed, insert, source = helpers.feed, helpers.insert, helpers.source
+local clear, execute, expect = helpers.clear, helpers.execute, helpers.expect
+
+describe('search_mbyte', function()
+  setup(clear)
```

Comment:
Usually I recommend `before_each(clear)` so that if new `it()` blocks are added in the future, it won't confuse people. (`setup()` runs only once per `describe()` block, whereas `before_each()` runs before each `it()`).